blocks/HARDWARE/PROTOCOLS/CAN/UTILS/DECODE/DECODE.py

```diff
@@ -0,0 +1,40 @@
+from flojoy import flojoy, Stateful, DataFrame
+import pandas as pd
+import logging
+
+
+@flojoy(deps={"python-can": "4.3.1", "cantools": "39.4.2"})
+def DECODE(
```

Comment:
I'd change this to DECODE_CAN or something because we will probably have a few other blocks, like DECODE_I2C.